frontend/compositions/profile-type/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+import Link from "next/link"
+import { profileTypeContent, publicUser, UserDataType } from "../../models/profile"
+import { useAuth, requireAuth } from "../../helpers"
+import styles from "./profile-type.module.css"
+
+export default requireAuth(function ProfileType() {
```

Comment:
The requireAuth call should wrap the whole Profile page component, so please move to `pages/profile/index.tsx`.

Reply:
got it, thanks